pydhsfw/tcpip.py

```diff
@@ -247,8 +247,12 @@ def _disconnect(self):
                 self._stream_reader.socket = None
                 self._stream_writer.socket = None
                 if sock:
-                    sock.shutdown(socket.SHUT_RDWR)
-                    sock.close()
+                    try:
+                        sock.shutdown(socket.SHUT_RDWR)
+                    except:
```

Comment:
I was thinking of being more specific on the `except OSError`, but that should work.

Reply:
OK. I think this is done. I'd like to merge.